www/faq.txt

```diff
@@ -91,13 +91,11 @@ the according (internal) command directly is called.
 
 Q: How can I let single clients float?
 --------------------------------------
-Not at all. You don't need it. You have the power of manual tiling, so
-there is no need to place clients manually by dragging. Even better:
-You don't even need to place them manually. Use a rule to place special
-dialogs automatically when they appear. See the GIMP-Example for a good
-example.
```

Comment:
Great catch! Maybe add a `Since herbstluftwm 0.8.0, …` at the beginning.

Reply:
Can be now?

Reply:
'Now' is imprecise, and I don't see the any disadvantage of being precise here :). If you don't want to start he answer with 'Since' you can also add `(since 0.8.0)` at the end of the answer.

Reply:
I meant now = after the latest commit. Though it's also not completely precise…

Reply:
Now it's a bit misleading. Now, it sounds like the shortcut method also works for hlwm older than 0.8.

Reply:
And now it sounds like a history lesson :-P

Reply:
I think it's good now, thanks!